src/alfasim_score/converter/alfacase/convert_alfacase.py

```diff
@@ -0,0 +1,49 @@
+from alfasim_sdk import AnnulusDescription
+from alfasim_sdk import FormationDescription
+from alfasim_sdk import ProfileDescription
+from alfasim_sdk import WellDescription
+from alfasim_sdk import XAndYDescription
+from barril.units import Scalar
+
+from alfasim_score.constants import ANNULUS_TOP_NODE_NAME
+from alfasim_score.constants import WELLBORE_BOTTOM_NODE
+from alfasim_score.constants import WELLBORE_TOP_NODE
+from alfasim_score.converter.alfacase.score_input_reader import ScoreInputReader
+from alfasim_score.units import LENGTH_UNIT
+
+
+class AlfacaseConverter:
+    def __init__(self, score_reader: ScoreInputReader):
+        self.score_input = score_reader
+        self.well_name = score_reader.score_filepath.stem
+
+    def _get_well_start_vertical_position(self) -> Scalar:
+        _, y = self.score_input.read_well_trajectory()
+        return Scalar(y[0], LENGTH_UNIT)
```

Comment:
Do not get it from trajectory, in json file we have water_depth and air_gap.
Get `-water_depth`.

Reply:
Shouldn't it be `-(water + air_gap)` ?

Reply:
Cota does not consider air gap. Only TVD and MD.

Reply:
Moreover, I belive the method could be rename to `_get_well_start_cota`.
However, it seems we will not need it anymore.

Reply:
Ok I forget we are gonna use zero as reference. Removed the method and now reference is zero.